lib/api/model/model.dart

```diff
@@ -275,80 +286,79 @@ class ZulipStream {
   Map<String, dynamic> toJson() => _$ZulipStreamToJson(this);
 }
 
-/// As in `subscriptions` in the initial snapshot.
+/// Policy for which users can post to the stream.
 ///
-/// For docs, search for "subscriptions:"
-/// in <https://zulip.com/api/register-queue>.
-@JsonSerializable(fieldRename: FieldRename.snake)
-class Subscription {
-  // First, fields that are about the stream and not the user's relation to it.
-  // These are largely the same as in [ZulipStream].
-
-  final int streamId;
-  final String name;
-  final String description;
-  final String renderedDescription;
-
-  final int dateCreated;
-  final int? firstMessageId;
-  final int? streamWeeklyTraffic;
-
-  final bool inviteOnly;
-  final bool? isWebPublic; // TODO(server-??): doc doesn't say when added
-  final bool historyPublicToSubscribers;
-  final int? messageRetentionDays;
-  // final List<int> subscribers; // we register with includeSubscribers false
+/// For docs, search for "stream_post_policy"
+/// in <https://zulip.com/api/get-stream-by-id>
+@JsonEnum(valueField: 'apiValue')
+enum StreamPostPolicy{
+  any(apiValue: 1),
+  admin(apiValue: 2),
+  member(apiValue: 3),
+  moderator(apiValue: 4),
+  unknown(apiValue: null);
 
-  final int streamPostPolicy; // TODO enum
-  // final bool? isAnnouncementOnly; // deprecated; ignore
-  final String emailAddress;
+  const StreamPostPolicy({
+    required this.apiValue,
+  });
 
-  final int? canRemoveSubscribersGroupId; // TODO(server-6)
+  final int? apiValue;
 
-  // Then, fields that are specific to the subscription,
-  // i.e. the user's relationship to the stream.
+  int? toJson() => apiValue;
+}
 
-  final bool? desktopNotifications;
-  final bool? emailNotifications;
-  final bool? wildcardMentionsNotify;
-  final bool? pushNotifications;
-  final bool? audibleNotifications;
+/// As in `subscriptions` in the initial snapshot.
+///
+/// For docs, search for "subscriptions:"
+/// in <https://zulip.com/api/register-queue>.
+@JsonSerializable(fieldRename: FieldRename.snake)
+class Subscription extends ZulipStream {
+  List<int>? subscribers;
 
-  final bool pinToTop;
+  bool? desktopNotifications;
+  bool? emailNotifications;
+  bool? wildcardMentionsNotify;
+  bool? pushNotifications;
+  bool? audibleNotifications;
 
-  final bool isMuted;
+  bool pinToTop;
+  bool isMuted;
   // final bool? inHomeView; // deprecated; ignore
 
-  final String color;
+  String color;
+  String emailAddress;
```

Comment:
Here we didn't stick to docs to maintain ordering for fields, and have grouped by data type, so it got moved with the other `String`. But removed per the change in FL 266.

Reply:
I see. I wouldn't describe the fields as grouped by data type — in main they look like this:
```dart
  // First, fields that are about the stream and not the user's relation to it.
  // These are largely the same as in [ZulipStream].

  // …
  // final List<int> subscribers; // we register with includeSubscribers false

  final int streamPostPolicy; // TODO enum
  // final bool? isAnnouncementOnly; // deprecated; ignore
  final String emailAddress;

  final int? canRemoveSubscribersGroupId; // TODO(server-6)

  // Then, fields that are specific to the subscription,
  // i.e. the user's relationship to the stream.
```

So `emailAddress` was in amidst `subscribers` and `streamPostPolicy` and others because they're fields that are about the stream and not the user's relation to it. If we were keeping it, I'd probably want it still next to `subscribers` for the same reason.

Moot because we're deleting it, though.